README.md

```diff
@@ -68,6 +68,19 @@ be highlighted.
 
     pip install Pygments
 
+If you don't have `pip` installed yet, you'll first need to install it.
+
+On Mac OS you can do so by installing Python with homebrew
```

Comment:
In before someone else notes the brand guidelines:

- `macOS`
- `Homebrew`

Also: This is a great change. Thanks for keeping the lights on!